src/index.js

```diff
@@ -0,0 +1,114 @@
+
+  const divTaskContent = document.createElement('div');
+  divTaskContent.classList.add('align-center', 'task-content');
+  const checkBok = document.createElement('input');
+  checkBok.type = 'checkbox';
+  checkBok.name = task[ind - 1].index;
+
+  const inTaskDescription = document.createElement('input');
+  inTaskDescription.classList.add('task-description');
+  inTaskDescription.value = task[ind - 1].description;
+  inTaskDescription.type = 'text';
+
+  checkBok.addEventListener('change', (event) => {
+    const { id } = event.currentTarget.parentElement.parentElement;
+    if (event.currentTarget.checked) {
+      event.currentTarget.nextSibling.style.textDecoration = 'line-through';
+      this.list[id - 1].completed = true;
+    } else {
+      event.currentTarget.nextSibling.style.textDecoration = 'none';
+      this.list[id - 1].completed = false;
+    }
+  });
+
+  if (task.completed) {
+    checkBok.click();
+    inTaskDescription.style.textDecoration = 'line-through';
+  }
+
+  inTaskDescription.addEventListener('focus', (event) => {
+    const { id } = event.currentTarget.parentElement.parentElement;
+    document.getElementById(id).style.backgroundColor = 'rgb(156, 156, 255)';
+    const iDelete = event.currentTarget.parentElement.nextSibling;
+    const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
+    iDelete.classList.remove('hidden');
+    iMenu.classList.add('hidden');
+  });
+
+  inTaskDescription.addEventListener('focusout', (event) => {
+    const { id } = event.currentTarget.parentElement.parentElement;
+    const litask = document.getElementById(id);
+    litask.style.backgroundColor = 'white';
+
+    this.list[id - 1].description = event.currentTarget.value;
+    const iDelete = event.currentTarget.parentElement.nextSibling;
+    const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
+
+    setTimeout(() => {
+      iMenu.classList.remove('hidden');
+      iDelete.classList.add('hidden');
+    }, 200);
+  });
+
+  divTaskContent.append(checkBok);
+  divTaskContent.append(inTaskDescription);
+
+  const iMenu = document.createElement('i');
+  iMenu.classList.add('fa-solid', 'fa-ellipsis-vertical');
+
+  const iDelete = document.createElement('i');
+  iDelete.classList.add('fa-solid', 'fa-trash', 'hidden');
+  iDelete.style.color = '#fff';
+  iDelete.addEventListener('click', (event) => {
+    const { id } = event.currentTarget.parentElement;
+    this.delete(id);
+  });
+
+  liTask.append(divTaskContent);
+  liTask.append(iDelete);
+  liTask.append(iMenu);
+
+  ulListTask.append(liTask);
+}
+
+/* On submit the input */
+const task = [];
+const form = document.forms[0];
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (form.newtask.value.trim().length !== 0) {
+    const item = { description: form.newtask.value, completed: false, index: task.length + 1 };
+    task.push(item);
+    display(task, item.index);
+  }
+  form.newtask.value = '';
+});
```

Comment:
- [ ] I like that you're thinking ahead and adding features but for this milestone, the requirement is that you just need to create a temporary array of todo task objects and render them by iterating.
- Requirement:
  ![image](https://user-images.githubusercontent.com/104968236/235934318-e3521abc-f775-4add-bb68-f2ea81ec2418.png)

- Check the example below:
```
const tasks = [
  {
    description: 'task-1',
    completed: false,
    index: 1,
  },
  {
    description: 'task-2',
    completed: false,
    index: 2,
  },
];
```
- And then you'll have to iterate and render each task.

Reply:
- manual insert functionality removed
- population of list automatically done